src/InboundServer/api.yaml

```diff
@@ -2604,15 +2604,15 @@ paths:
           $ref: '#/components/responses/ErrorResponse501'
         503:
           $ref: '#/components/responses/ErrorResponse503'
-  /thirdPartyRequest/transactions/{ID}/error:
+  /thirdpartyRequest/transactions/{ID}/error:
```

Comment:
this should be /thirdpartyRequests/transactions/{ID}/error:

Reply:
done. thank you good spot.